graph/ops_convolve_test.go

```diff
@@ -240,3 +240,29 @@ func TestGradientConvolve(t *testing.T) {
 			tensors.FromScalarAndDimensions(0.0, 2, 2, 3, 6).Value(),
 		})
 }
+
+func TestConvolveForwardPass(t *testing.T) {
+	testFuncOneInput(t, "Convolve identity kernel with FeatureGroupCount and BatchGroupCount",
+		func(g *Graph) (input, output *Node) {
+			// Create 3x3 input with channels first (batch=1, channels=1)
+			input = Ones(g, MakeShape(dtypes.Float32, 1, 1, 3, 3))
```

Comment:
Would you mind using instead of `Ones` with the shape, use `IotaFull` with the same shape ? This makes the test slightly more robust...